src/lib/integration_test_lib/wait_condition.ml

```diff
@@ -297,12 +299,15 @@ struct
         ~metadata:
           [ ("state_hash", State_hash.to_yojson breadcrumb_added.state_hash)
           ; ("txn_hash", Mina_transaction.Transaction_hash.to_yojson txn_hash)
-          ; ("txn", Zkapp_command.to_yojson zkapp_command)
+          ; ("txn", Zkapp_command.Stable.Latest.to_yojson zkapp_command)
           ] ;
       [%log' debug (Logger.create ())]
         "wait_condition check, zkapp_to_be_included_in_frontier, \
          zkapp_command: $zkapp_command "
-        ~metadata:[ ("zkapp_command", Zkapp_command.to_yojson zkapp_command) ] ;
+        ~metadata:
+          [ ( "zkapp_command"
+            , Zkapp_command.Stable.Latest.to_yojson zkapp_command )
```

Comment:
Log format change in integration test. Not a structured log, hence shouldn't affect anything.